### PR TITLE
See the content of the body field which is surrounding the first matching search term [CPCN-18]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: assets/wire/utils.js [hunk 1]
```diff
@@ -186,6 +186,27 @@ export function shortText(item, length=40, config) {
     return words.slice(0, length).join(' ') + (words.length > length ? '...' : '');
 }
 
+export function shortHighlightedtext(text, maxLength = 40) {
+    const spanRegex = /<span\s+class="es-highlight">([^<]+)<\/span>/i;
+    const match = text.match(spanRegex);
+  
+    if (match) {
+        const startIndex = text.indexOf(match[0]);
```
Comment (petrjasek) [inline assets/wire/utils.js:194]: isn't this the same as `match.index` ?

**Review by petrjasek — COMMENTED** [review]
can you provide some samples, ideally write some tests so it's obvious what it does.
imo it won't take the text before the matching string, but that was the idea to get a few words before and after so there is more context.

File: assets/wire/utils.js (hunk 1 above)
Comment (devketanpro) [reply assets/wire/utils.js:194]: yeah you are right !!

**Comment by devketanpro** [comment]
currently with this changes it looks like this :- 
image

**Comment by petrjasek** [comment]
I think this would be better done on some html tree level, you might get some broken html if you handle that as a text

**Comment by devketanpro** [comment]
now with these changes it looks like this :- 
image

File: assets/wire/tests/WireApp.spec.js [hunk 2]
```diff
@@ -57,3 +58,32 @@ describe('WireApp', () => {
         expect(actions.length).toBe(3);
     });
 });
+
+
+describe('shortHighlightedtext', () => {
+    it('returns truncated text with highlighted span', () => {
```
Comment (petrjasek) [inline assets/wire/tests/WireApp.spec.js:64]: can you also add a test case for list and table?

**Review by petrjasek — APPROVED** [review]

**Pull request merged**